testData/folding/switchSelectCaseStatement.go

```diff
@@ -0,0 +1,41 @@
+package foo
+
+func main() <fold text='{...}'>{
+	for <fold text='{...}'>{
+		println()
+	}</fold>
+	
+	sum := 0
+	for i := 0; i < 10; i++ <fold text='{...}'>{
+		sum += i
+	}</fold>
+
+	ch := make(chan int)
+	select <fold text='{...}'>{
+	case _ <- ch:
+		println("demo")
+	case _ <- ch:<fold text='{...}'>
+		println("demo")
+		println("demo")</fold>
+	}</fold>
+
+	switch true <fold text='{...}'>{
+	case 1 == 2:
+		println("demo")
+
+	case 2 == 3:<fold text='{...}'>
+		println("demo")
+		println("demo")</fold>
+
+	default:<fold text='{...}'>
```

Comment:
I felt this is more appropriate than `...` as in Go it's optional to use the curly braces for the case / default statements. But can change it if it makes more sense.